### DISCUSSION TITLE
Ads not being blocked on 1.1.69.612

### DISCUSSION THREAD

**Discussion by desk-job** [SpotX-Official/SpotX#12] [Errors]
Not sure what's wrong but none of the ads are being blocked on the newest version of Spotify (1.1.69.612). I uninstalled and re-installed multiple times. Running the Powershell command just downloads an older version and then when I update it, the ads are back (even after I run the patch).

**Comment by amd64fox** [comment]
What type of advertising do you have ? Can I have a screenshot ?

**Reply by desk-job** [reply]
All kinds of ads. This one is on the home page: 
![home](https://user-images.githubusercontent.com/71421612/135570701-baae2d6f-ac52-45d4-ae71-66050f35f40e.png)
 
This one was while I was playing an album. It's got the banner in the centre of the client and the audio ad playing in the background:
![banner + audio](https://user-images.githubusercontent.com/71421612/135570798-e89c00ce-ae58-402f-bada-56c4e6baf195.png)
 
Version: 
![image](https://user-images.githubusercontent.com/71421612/135570867-916ded88-45ea-4834-a674-1e20c26c89d8.png)

**Reply by amd64fox** [reply]
BTS patch stopped working stably in new version.

I advise you to temporarily roll back to the previous, working version for you and block updates.

Waiting for BTS to be repaired ...

**Reply by amd64fox** [reply]
Bts has been fixed
